web/ui/react-app/src/components/SearchBar.tsx

```diff
@@ -23,7 +24,7 @@ const SearchBar: FC<SearchBarProps> = ({ handleChange, placeholder }) => {
       <InputGroupAddon addonType="prepend">
         <InputGroupText>{<FontAwesomeIcon icon={faSearch} />}</InputGroupText>
       </InputGroupAddon>
-      <Input autoFocus onChange={handleSearchChange} placeholder={placeholder} />
+      <Input autoFocus onChange={handleSearchChange} placeholder={placeholder} defaultValue={defaultValue} />
```

Comment:
Right now, if you load a page with a search term set, the search is not executed and the page will show all data, unfiltered. Probably we need to always execute `handleSearchChange` once upon load of the search bar component, at least when the search term is not empty?

Reply:
ready! ;)